src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,82 @@
+import classNames from 'classnames';
+import React from 'react';
+
+interface Props {
+  todosCount: number;
+  completedCount: number;
+  filter: 'all' | 'active' | 'completed';
+  setFilter: (filter: 'all' | 'active' | 'completed') => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  todosCount,
+  completedCount,
+  filter,
+  setFilter,
+}) => {
+  const handleFilterChange = (newFilter: 'all' | 'active' | 'completed') => {
+    setFilter(newFilter);
+  };
+
+  const getFilteredTodosCount = () => {
+    switch (filter) {
+      case 'all':
+        return todosCount;
+      case 'active':
+        return todosCount - completedCount;
+      case 'completed':
+        return completedCount;
+      default:
+        return completedCount;
```

Comment:
```suggestion
      case 'completed':
      default:
        return completedCount;
```
jou may join a few cases in this way